pkgs/sagemath-standard/sage_setup

```diff
@@ -0,0 +1 @@
+../../src/sage_setup
```

Comment:
please remove this change

Reply:
I didn't get a chance to explain. The symlink is there so building the sdist actually works just with setuptools.

Without this symlink, there's no way to build the sdist:
```
$ VENV_FOR_SDIST/bin/python -m build -snx
...
ModuleNotFoundError: No module named 'sage_setup'
```
or worse:
```
$ VENV_FOR_SDIST/bin/python -m build -sx
* Creating venv isolated environment...
* Installing packages in isolated environment... (cypari2 >=2.1.1, cysignals >=1.10.2, cython >=3.0, != 3.0.3, <4.0, gmpy2 ~=2.1.b999, jinja2 >=3.0, jupyter_core >=4.6.3, memory_allocator, numpy >=1.19, pkgconfig, pplpy >=0.8.6, sage-conf ~= 10.3b7, sage-setup ~= 10.3b7, sage_setup[autogen], setuptools >= 68.1.1, wheel >=0.36.2)
...
ERROR: Could not find a version that satisfies the requirement sage-setup~=10.3b7 (from versions: 9.4rc1)
ERROR: No matching distribution found for sage-setup~=10.3b7
```

Reply:
Obviously you need to make the distribution sage-setup in the correct version available.

Reply:
ok, I removed the symlink.